src/Core/GraphConstants.php

```diff
@@ -23,7 +23,7 @@ final class GraphConstants
     const REST_ENDPOINT = "https://graph.microsoft.com/";
 
     // Define HTTP request constants
-    const SDK_VERSION = "1.28.0";
+    const SDK_VERSION = "1.29.0";
```

Comment:
I believe this should still be 1.28.0? Although I see we have a 1.29.0 on Packagist and I'm not sure how that got there.

Reply:
Yep, I made a mistake and didn't realize that I had already bumped the number but hadn't released. I tagged it before pushing to main which resulted in automatic publish to packagist.

Reply:
Cool, makes sense